packages/react-form/src/types.ts

```diff
@@ -72,6 +74,7 @@ export interface Form<T extends FieldBag> {
   reset(): void;
   submit(event?: React.FormEvent): void;
   makeClean(): void;
+  lists?: ListBag;
```

Comment:
I am not sure if we should rename to `DynamicListBag` because we can possibly in future create a new type of list and want to do the exact same thing for that list. The way it is now, you can possibly add another type of list in the bag.